src/utils/constants.ts

```diff
@@ -1,3 +1,6 @@
+// THREE
+export const ASPECT_RATIO = (): number => window.innerWidth / window.innerHeight;
```

Comment:
Why did you make `ASPECT_RATIO ()` a function instead of just a simple constant `ASPECT_RATIO `?

Reply:
https://github.com/BryanPan342/e-motion/pull/13#discussion_r644476760